packages/@glimmer/compiler/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@glimmer/compiler",
   "version": "0.84.3",
+  "type": "module",
   "repository": "https://github.com/glimmerjs/glimmer-vm/tree/master/packages/@glimmer/compiler",
   "exports": {
     "types": "./index.ts",
     "development": "./index.ts",
```

Comment:
should development be pointing at ts here? is this `development` meant for in-repo, stuff? or would it be seen by consumers? Feels goofy that `development` would require the consumer to have a ts compiler set up on their node_modules? Maybe i misunderstand something though?